### PR TITLE
docs: The return type of the stop function corrected to as boolean type

### PR DESCRIPTION
**Pull request by VIRAT9358**
#### Checklist

Expected Behavior Now
The return type of the stop function should be correctly typed as boolean
 
- changes added at https://github.com/VIRAT9358/electron/blob/my-branch/docs/api/power-save-blocker.md

#### Release Notes

Notes: none

### CODE REVIEW

**Review by codebytere — CHANGES_REQUESTED** [review]
This example code is not Typescript and should not have types. Can you please explain further what you were trying to do?

**Review by codebytere — COMMENTED** [review]
@VIRAT9358 we'd be happy to help this move forward if possible - what you need to do to correct this is change https://github.com/electron/electron/pull/39080/files#diff-2fad1de29e653bd23d81332c4a19bd7a28094897557c7e02221b277fff684d02R55-R60 to:

```md
### `powerSaveBlocker.stop(id)`

* `id` Integer - The power save blocker id returned by `powerSaveBlocker.start`.

Returns `boolean` - Whether the specified `powerSaveBlocker` has been stopped.
```

**Comment by VIRAT9358** [comment]
Ok I will try to it today.Sorry I misinterpreted the issue

**Comment by VIRAT9358** [comment]
const { powerSaveBlocker } = require('electron')

/**
 * Starts a power save blocker with the given type.
 * @param {string} type - The type of power save blocker to start.
 * @returns {number} - The power save blocker id.
 */
const startPowerSaveBlocker = (type) => {
  return powerSaveBlocker.start(type);
};

/**
 * Stops the specified power save blocker.
 * @param {number} id - The power save blocker id returned by `powerSaveBlocker.start`.
 * @returns {boolean} - Whether the specified `powerSaveBlocker` has been stopped.
 */
const stopPowerSaveBlocker = (id) => {
  const isStopped = powerSaveBlocker.stop(id);
  return isStopped;
};

const id = startPowerSaveBlocker('prevent-display-sleep');
console.log(stopPowerSaveBlocker(id));
Can u validate this

**Comment by VIRAT9358** [comment]
`const { powerSaveBlocker } = require('electron')

/**
 * Starts a power save blocker with the given type.
 * @param {string} type - The type of power save blocker to start.
 * @returns {number} - The power save blocker id.
 */
const startPowerSaveBlocker = (type) => {
  return powerSaveBlocker.start(type);
};

/**
 * Stops the specified power save blocker.
 * @param {number} id - The power save blocker id returned by `powerSaveBlocker.start`.
 * @returns {boolean} - Whether the specified `powerSaveBlocker` has been stopped.
 */
const stopPowerSaveBlocker = (id) => {
  const isStopped = powerSaveBlocker.stop(id);
  return isStopped;
};

const id = startPowerSaveBlocker('prevent-display-sleep');
console.log(stopPowerSaveBlocker(id));`

**Comment by VIRAT9358** [comment]
In the modified code, I've created two separate functions, startPowerSaveBlocker and stopPowerSaveBlocker, which handle the starting and stopping of power save blockers, respectively. The startPowerSaveBlocker function returns the power save blocker id as before, and the stopPowerSaveBlocker function returns a boolean value indicating whether the specified power save blocker has been stopped or not.

**Comment by VIRAT9358** [comment]
```javascript
const { powerSaveBlocker } = require('electron')

/**
 * Starts a power save blocker with the given type.
 * @param {string} type - The type of power save blocker to start.
 * @returns {number} - The power save blocker id.
 */
const startPowerSaveBlocker = (type) => {
  return powerSaveBlocker.start(type);
};

/**
 * Stops the specified power save blocker.
 * @param {number} id - The power save blocker id returned by `powerSaveBlocker.start`.
 * @returns {boolean} - Whether the specified `powerSaveBlocker` has been stopped.
 */
const stopPowerSaveBlocker = (id) => {
  const isStopped = powerSaveBlocker.stop(id);
  return isStopped;
};

const id = startPowerSaveBlocker('prevent-display-sleep');
console.log(stopPowerSaveBlocker(id));


```

**Comment by VIRAT9358** [comment]
> ```js
> const { powerSaveBlocker } = require('electron')
> 
> /**
>  * Starts a power save blocker with the given type.
>  * @param {string} type - The type of power save blocker to start.
>  * @returns {number} - The power save blocker id.
>  */
> const startPowerSaveBlocker = (type) => {
>   return powerSaveBlocker.start(type);
> };
> 
> /**
>  * Stops the specified power save blocker.
>  * @param {number} id - The power save blocker id returned by `powerSaveBlocker.start`.
>  * @returns {boolean} - Whether the specified `powerSaveBlocker` has been stopped.
>  */
> const stopPowerSaveBlocker = (id) => {
>   const isStopped = powerSaveBlocker.stop(id);
>   return isStopped;
> };
> 
> const id = startPowerSaveBlocker('prevent-display-sleep');
> console.log(stopPowerSaveBlocker(id));
> ```

In the modified code, I've created two separate functions, startPowerSaveBlocker and stopPowerSaveBlocker, which handle the starting and stopping of power save blockers, respectively. The startPowerSaveBlocker function returns the power save blocker id as before, and the stopPowerSaveBlocker function returns a boolean value indicating whether the specified power save blocker has been stopped or not.

**Comment by codebytere** [comment]
@VIRAT9358 to clarify, we are _not_ changing the example. We are _only_ changing the documentation itself, as i noted above.

File: docs/api/power-save-blocker.md [hunk 1]
```diff
@@ -7,12 +7,21 @@ Process: [Main](../glossary.md#main-process)
...
+// console.log(powerSaveBlocker.isStarted(id))
+
+// powerSaveBlocker.stop(id)
+const { powerSaveBlocker } = require('electron');
+
+const id = powerSaveBlocker.start('prevent-display-sleep');
+console.log(powerSaveBlocker.isStarted(id));
+
+const stopped = powerSaveBlocker.stop(id) as unknown as boolean;
+console.log(stopped);
```
Comment (codebytere) [inline docs/api/power-save-blocker.md:22]: none of this should be changed

**Pull request closed without merging**